src/webapp/models.py

```diff
@@ -204,6 +205,28 @@ def get_contacts_data(self) -> ContactsData:
 
         return self.merged_contacts_data.data
 
+    def get_ligo_dns(self) -> List[str]:
+        """
+        Get list of DNs of authorized LIGO users from their LDAP
+        """
+        if not (self.ligo_ldap_url and self.ligo_ldap_user and
+                self.ligo_ldap_passfile):
+            log.debug("LIGO_LDAP_{URL|USER|PASSFILE} not specified; "
+                      "getting empty list")
```

Comment:
i would say `"returning empty list"`, but i see there is precedent elsewhere for the "getting" wording

Reply:
These are debug messages so if we think "returning" is better then we can change it in multiple places -- but that's a separate PR :)